pkgs/applications/editors/jupyter-kernels/clojupyter/deps.nix

```diff
@@ -1105,3 +1105,4 @@
 
   ];
   }
+  
```

Comment:
I think this extra blank line is breaking the Editor Config checks.

Reply:
Good catch -- emacs config added that. Will revise.

Reply:
BTW, not really related to the review itself but did you generate this file by hand 🤔 ?

Reply:
I did not! I just ran `update.sh`, which seems like it just invokes `clj2nix`. I know there's been a lot of churn in different clojure nix tools but I figured no need to reopen that can of worms for just a version bump!